Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -647,7 +647,7 @@ private async Task<string> GetToken(ListingsProviderInfo info, CancellationToken
         {
             using var options = new HttpRequestMessage(HttpMethod.Post, ApiUrl + "/token");
             var hashedPasswordBytes = _cryptoProvider.ComputeHash("SHA1", Encoding.ASCII.GetBytes(password), Array.Empty<byte>());
-            string hashedPassword = Hex.Encode(hashedPasswordBytes);
+            string hashedPassword = Convert.ToHexString(hashedPasswordBytes);
```

Comment:
```suggestion
            string hashedPassword = Convert.ToHexString(hashedPasswordBytes).ToLower();
```
SchedulesDirect requires lowercase hex